src/reversion/models.py

```diff
@@ -214,3 +194,6 @@ def __str__(self):
     #Meta
     class Meta:
         app_label = 'reversion'
+        index_together = [
+            ["content_type", "object_id"],
```

Comment:
The only downside to that is then we would need an index on content_type for FK validation.

All your internal logic always specifies the content_type, but I understand others might not expect that situation.  (Though it would be dangerous to do that lookup and not specify the content_type, since the object_id is not guaranteed to be unique across content_types.)

If you still want me to switch it and re-add the FK, it's fine with me.